client/js/libs/handlebars/ircmessageparser/merge.js

```diff
@@ -0,0 +1,60 @@
+"use strict";
+
+const anyIntersection = require("./anyIntersection");
+const fill = require("./fill");
+
+let Object_assign = Object.assign;
+
+if (typeof Object.assign !== "function") {
```

Comment:
```
if (typeof Object_assign !== "function")
```
or ditch the `Object_assign` and just assign to `Object.assign`
iirc the Object_assign was not the according polyfill and just an incorrect implementation that was "good enough"